index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Friends Fansite</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/index.css">
+  <link rel="stylesheet" href="../styles/forms.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul id="header-links">
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="../pages/gallery.html">Gallery</a></li>
+        <li><a href="../pages/facts.html">Facts</a></li>
+      </ul>
+    </nav>
+    <div class="fadeIn">
+      <img id="friends-logo" src="../images/Friends_logo.svg" alt="Friends Logo">
+      <h1 id="subtitle">
+        F<span>&middot;</span>a<span>&middot;</span>n<span>&middot;</span>s<span>&middot;</span>i<span>&middot;</span>t<span>&middot;</span>e
+      </h1>
+    </div>
+  </header>
+  <main>
+    <div class="container fadeIn">
+      <img src="../images/accent_image.webp" alt="Friends Cast" class="accent_img">
+      <section>
+        <h2>Welcome <span>&middot;</span> to <span>&middot;</span> our <span>&middot;</span> friends <span>&middot;</span> fansite!</h2>
+        <p>
+          <i><b>Friends</b></i> is an American television sitcom created by
```

Comment:
The [i tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) per the docs "represents a range of text that is set off from the normal text for some reason, such as idiomatic text, technical terms, taxonomical designations". The [b tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) "should not use b tag for styling text or granting importance. If you wish to create boldface text, you should use the CSS". 

Prefer that _"**Friends**"_ gets styled by CSS.